src/Files.App.Controls/ThemedIcon/Data/ThemedIconToggleBehaviors.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Controls
+{
+	/// <summary>
+	/// Defines IconTypes for <see cref="ThemedIcon"/>.
+	/// </summary>
+	public enum ToggleBehaviors
+	{
+		/// <summary>
+		/// Toggle Behavior type of <see cref="ThemedIcon"/> is Auto.
+		/// </summary>
+		Auto,
```

Comment:
I'm a bit puzzled what the auto behavior is supposed to do.

Reply:
At present, when **Auto** is set, if the ThemedIcon is within a ToggleButton, that "owner" control's IsChecked state will set the Icon into the correct Toggled and Filled states.

But if we want to control it differently, we need a way to manually set this, so we check if the **ToggleBehavior** is Auto, and respond to it.  ToggleBehavior can also be On and Off, to manually set the states.

Reply:
It's a mixed Enum for IgnoreOwnerToggle and IsToggled.

Auto: IgnoreOwnerToggle=true, IsToggled=null
On: IsToggled=true
Off: IgnoreOwnerToggle=false

Reply:
Absolutely true.
Let's change comments.